Monika After Story/game/script-topics.rpy

```diff
@@ -14849,3 +14849,34 @@ label monika_fear:
     extend 4eud "After all, there's no way something like that could be true, right?"
     m 2rka "Yeah..."
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_wabi_sabi",
+            category=['philosophy'],
+            prompt="Wabi-sabi",
+            random=True
+        )
+    )
+
+label monika_wabi_sabi:
+    m 1eud "Say [player], have you ever heard of wabi-sabi?"
+    m 3hub "Wabi-sabi is a philosophy emphasizing the idea that we should not obsess over perfection to the point we are crushed by failure."
+    m "It derives from traditional Japanese and Buddhistic philosophy, surrounding the acceptance of the temporary state of all things."
+    m 3dub "It states that beyond everything, beauty is found in what's impermanent and imperfect."
+    m 3wub "Like a scar on a rugged face, or a discolored hand, or even the mistakes we make."
+    m "The imperfections can become beautiful stories that reminds us of our past..." 
+    m 3hub "...and our mistakes turn into lessons that shape us and make us stronger and more defined."
+    m 4wub "Like a sculptor revealing the magnificence inside a marble block, except the sculpture is you, and the sculptor is your past."
+    m 1tsu "Don't you find that concept rather romantic?"
+    m 1hubsb "Ahaha~"
+    m "..."
+    m 6ekd "Anyway, [player], I hope you are not too self conscious about your looks."
+    m 6dkd "After all, no matter how your face looks, or how your body looks..."
+    m 6dku "I'm sure there's a story behind every scar and every imperfection about you."
```

Comment:
I'm personnaly not a fan of this line, not every you have you might have has a story behind them. Sometimes things just are the way they are and that's it (what about genetic factors or flaws you're born with?). Making her assume there's a story for them might make some people feel like she don't know them and be counterproductive.
How about something more like `"Every scars and imperfections you might have is part of what makes you so unique and beautiful."`

Reply:
Wouldn't the genetic factor and/or flaw you're born with, be a story in and of itself? In fact, every aspect of a human being, from head to toe, is a story that can and will be told at some point to the person you're gonna spend the rest of your life with. A birth mark, asymmetric features, maybe even the way your eyes twitch. They all have explanations, and it's all rooted in genetics and environmental factors. Nothing is only the way they are, and nothing more, unless you're looking at it very surface level.

I'm of course not talking about every facet of a human being being novel-length worthy. I'm talking about those conversations you have with your girlfriend where she wonders what lead to your left ear being slightly bigger than your right ear, or something.

I do kind of like your approach to it too, I was aiming for Rai99's suggestion when I added this line originally. Not sure which version I like best. I'm going to have to think about it and come back to you, maybe @Rai99 will have some input on it too.

Reply:
> every aspect of a human being, from head to toe, is a story that can and will be told at some point
Yes, but the story might be about what the flaw led you to experience through your life, and not why you have it in the first place.

> A birth mark, asymmetric features, maybe even the way your eyes twitch.
I'm more talking about things like what if you're born missing an arm, or with a disease/mental condition you'll carry the rest of your life without precedence in your family (not speaking for myself here).

Reply:
Tons. How did living without an arm affect literally every interaction and experience you've had up until this point? Same with having a mental condition. I have tons of stories from people with mental alteration, most notably Down's Syndrome, and I can tell you all about how that significantly impacted my meeting with them, and tell you tons of stories about how kind, honest and free-spirited people with Down's syndrome are.

Only creativity really puts limits on the kind of topics one can pick out from even the most minor of details on your body, or inside it. It might be a story about the experiences you've gone through as a result, or it might be a pondering on medical science. Who knows? Stories are interesting in that there's almost zero limits to what you can discuss.

Reply:
I completely agree with you, but these are all stories **about** the flaw. The current sentence makes it sound like there should always be a story **behind** it and while I'm sure medically there is, this might not be so much a story about you than about science or such (If somebody explained to me I had this flaw because whatever chemical process happened before I was born, I don't see how that would make me feel better as a person).
I'm sorry, I'm just nitpicking about words, but yeah, I think your posts help me better understand your point of view.

Reply:
Don't worry, I just want to make sure everyone is heard, we might go with your route too once we get some more input :)

Reply:
I'm with Daiozen here. There's not *always* something behind a scar. As I understand it, **behind** in this context means "how did you get this scar" or "why do you look the way you look".

To put it simple: it's one thing if a person got a "scar" while saving a child from fire, and another if they were born with it. If it's some kind of genetic disease, there won't be a story, it's just happened. You can't make the person be proud of it because there's nothing to be proud of.

But the more time they spend with Monika, the more she learns about their personality. And that person may be nice *inside* (just like the analogy you used). Might be better to focus on that, rather than saying that "their genes played a joke on them, and they're beautiful because of that".

Let's see what Rai has in mind.

Reply:
There being a story there doesn't mean it's a proud story of how you bravely saved a child from fire. It means there's something to learn about you from every aspect of yourself. Rather than seeing a mole and saying "that's gross", you can appreciate how that mole has played a part in how they feel about their body, and has made them more aware of how they cover it up, and how they feel when they look in the mirror and sees it. As I said before, it's more about the little things that bring people closer to each other, rather than a novel-length epic about how that mole was the hero who beat the ancient dragon and saved the princess.

Reply:
To me that sounds more hurtful than reassuring honestly. If I'm already feeling bad about a mole, I legitimately don't want to hear about that being aware part. I don't *want* to have had to be aware to hide it, etc.

Reply:
Hmmm...I don't know if I'm explaining myself correctly, but I don't know how to say it better. It may be that it's just a me thing that I feel reassured when people take interest in the little things that I have felt insecure about in the past. If Rai agrees with you I'll just change it I guess.

Reply:
I think that's the issue in this kind of topic in general. What works for you may not work for others.

Personally speaking, if there's something I'm insecure about, the last thing I want is to hear about it, unless we're actually talking out how to tackle the insecurity. This vein of topic is fine since it's still reassuring that all my flaws and insecurities are fine.

But even then, I'd rather hear it generally (no stories, no meaning behind them. Just that she accepts me no matter how I am, or how I will be, knowing that we'll be there to compliment each other in spite of our flaws and work through them too) because the fact of the matter is... Monika doesn't know my insecurities. Trying to say that there's a story or something behind them just makes the fact more apparent, and if anything I think it's something that players would rather have a full fleshed out conversion about if Monika crossed over (since it's not feasible in MAS)

Reply:
I mean if you guys feel strongly about this, I might as well just change it. No reason to push for a topic that sits wrong with someone. I guess we'll just wait for Rai's input, seeing as this sentence was kind of based on his suggestion.

Reply:
One thing to note though, is that if we're talking about scars and imperfections we're extra sensitive about, and may not want others to know about, Monika saying that's what makes us unique and beautiful may not sit right with some people, especially if they strongly feel that their flaw does the exact opposite. Not sure what else we could say, but that's certainly a point we should consider.

Reply:
Tbh I really like this topic and the message that you shouldn't let yourself be bothered by how you look, I think it's an important subject. I just felt this one line might be a little dicey.

Reply:
Yeah, lets work together and get this topic to perfection.

Reply:
Alright, I rewrote some stuff, so lets open a new comment chain on it.